imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -85,25 +89,12 @@ class CaseExplorer extends Component {
                 </div>
                 {isExpanded && (
                   <ul className='list bg-light-gray ma0 pl0 shadow-in-top-1'>
-                    {unitsDict[unitTitle].map(caseItem => (
-                      <li key={caseItem.id} className='h2-5 bt b--black-10'>
-                        <div className='flex items-center'>
-                          <Link
-                            className={
-                              'link flex-grow ellipsis ml3 pl1 ' +
-                                (isClosed(caseItem) ? 'silver strike' : 'bondi-blue')
-                            }
-                            to={`/case/${caseItem.id}`}
-                            onClick={() => dispatch(storeBreadcrumb(match.url))}
-                          >
-                            {caseItem.title}
-                          </Link>
-                          <IconButton>
-                            <FontIcon className='material-icons' color={moreIconColor}>more_horiz</FontIcon>
-                          </IconButton>
-                        </div>
-                      </li>
-                    ))}
+                    <Caselist
+                      closedCases={closedCases}
+                      openCases={openCases}
+                      dispatch={dispatch}
+                      match={match}
```

Comment:
~~dispatch~~ and ~~match~~
Instead => `onItemClick={() => dispatch(storeBreadcrumb(match.url))}`